lms/envs/common.py

```diff
@@ -1610,7 +1610,20 @@
 # and defines some "args" function that returns a list with the command arguments to execute.
 REQUIRE_ENVIRONMENT = "node"
 
-
+# In production, the Django pipeline appends a file hash to JavaScript file names.
+# This makes it difficult for RequireJS to load its requirements, since module names
+# specified in JavaScript code do not include the hash.
+# For this reason, we calculate the actual path including the hash on the server
+# when rendering the page.  We then override the default paths provided to RequireJS
+# so it can resolve the module name to the correct URL.
+#
+# If you want to load JavaScript dependencies using RequireJS
+# but you don't want to include those dependencies in the JS bundle for the page,
+# then you need to add the module and URL path to this dictionary.
+REQUIRE_JS_PATH_OVERRIDES = {
```

Comment:
Nit: this would be simpler as an array, if it is always the case that the value is just the key with ``.js`` added to it. Are you thinking that there might be situations where the path would be very different?